README.md

```diff
@@ -50,6 +54,8 @@ pulling the latest version of `master` from the Github repository inside the `jw
     git pull origin master
     python setup.py develop
 
+### CRDS Setup ###
+
```

Comment:
It would be useful to add: 'Within the STScI network, the default configuration of CRDS can be used to run the pipeline.'

Reply:
oh yah, sorry, I forgot crds does keep a local cache, I always use a local cache.... but maybe we should point to the crds docs here: https://jwst-crds.stsci.edu/static/users_guide/environment.html

Reply:
I added the "default config" info.  I don't want to point to CRDS docs here, as I want to keep this as short and simple as possible.  In fact, the README is already too long.  =)